handlers/test.go

```diff
@@ -59,6 +59,7 @@ type SendPrepFunc func(*httptest.Server, courier.ChannelHandler, courier.Channel
 type MockedRequest struct {
 	Method       string
 	Path         string
+	RawQuery     string
```

Comment:
VK requests are made by parameters.

Reply:
Do you actually need this though? Isn't the path enough to differentiate between the different calls or am I misunderstanding that?

Reply:
Yes, I need.

A request to `"/messages.json?access_token=abc123xyz&message=Simple+message""` for example, will be a request with `URL.Path` equal to `"/messages.json"` and `URL.RawQuery` equal to `"access_token=abc123xyz&message=Simple+message"`.